service/src/main/java/bio/terra/drshub/models/DrsApi.java

```diff
@@ -17,4 +30,54 @@ public void setBearerToken(String bearerToken) {
   public void setHeader(String name, String value) {
     this.getApiClient().addDefaultHeader(name, value);
   }
+
+  @Override
+  public DrsObject postObject(Object body, String objectId) throws RestClientException {
```

Comment:
The main downside with this approach is that new APIs won't have this retry and we won't get a compile error when they're added. Changing this to embed `ObjectsApi` instead of subclassing is one way to fix this.